### DISCUSSION TITLE
Predict the TSS using a new Transcription binding Motif

### DISCUSSION THREAD

**Discussion by truong128** [jzhoulab/puffin#15] [Q&A]
Hi Everyone,

I wonder if Anyone can help to show me how to predict the TTS with a completely new Motif (TGAGTCA) using Puffin? I did try to download the source code to re-train the model using my new Motif (TGAGTCA) but I could not find those associated files although I already checked it here (https://github.com/jzhoulab/puffin_manuscript).

Thanks a lot for your help

Ryan

**Comment by jzthree** [comment]
Thanks for the question. This is in principle achievable but requires understanding of how the model training works. I recommend loading the final Puffin model, add a new conv and deconv layer with your new motifs and freeze all weights except for the new motif and train it.
